libobs/obs-hevc.h

```diff
@@ -30,6 +30,66 @@ EXPORT void obs_extract_hevc_headers(const uint8_t *packet, size_t size,
 				     uint8_t **header_data, size_t *header_size,
 				     uint8_t **sei_data, size_t *sei_size);
 
+enum {
```

Comment:
Do not expose enum in header if it's not public.

Reply:
Done.